gps/app/views/Settings.js

```diff
@@ -55,11 +54,45 @@ export const SettingsScreen = ({ navigation }) => {
     };
   }, [navigation]);
 
+  //  This method is used to track whether user has "Allow" or "Deny" location permission system Dialog.
+  const systemDialogPermission = async () => {
```

Comment:
I'm confused about what this method is doing. Is there a dialog that is being prompted?